apps/omni/src/index.ts

```diff
@@ -0,0 +1,53 @@
+import * as dotenv from 'dotenv'
+dotenv.config()
```

Comment:
maybe move this out of the imports block

Reply:
Done